config/routes.rb

```diff
@@ -12,7 +12,7 @@
     namespace :api do
       namespace :v1 do
         resources :categories, only: [:index]
-        resources :floofs, only: [:index, :show] do
+        resources :floofs, only: [:index, :show, :new, :create] do
```

Comment:
Can we remove this change from this particular PR since the API endpoints aren't created as a part of this PR?